tensorflow/lite/testing/op_tests/elementwise.py

```diff
@@ -77,7 +78,8 @@ def make_sin_tests(options):
 @register_make_test_function()
 def make_log_tests(options):
   """Make a set of tests to do log."""
-  return _make_elementwise_tests(tf.math.log)(options)
+  return _make_elementwise_tests(tf.math.log, allow_fully_quantize=True,
+                                 min_value=.1, max_value=1)(options)
```

Comment:
Why do you set the max value to 1 here?

Reply:
It's a bit arbitrary as I used the same range as the rsqrt test. I could leave it to the default 100 value if it's preferred.

Reply:
Yeah, please leave the default value. Everything else looks good though!

Reply:
Thanks, I updated it to 100.